README.md

```diff
@@ -27,315 +28,19 @@ Or from [GitHub](https://github.com/) with:
-#> [23] "  languagelinks: navbar"           "  languagecodes:"                 
-#> [25] "  - name: es"                      "    text: \"Version in es\""      
-#> [27] "  - name: fr"                      "    text: \"Version in fr\""      
-#> [29] "  - name: en"                      "    text: \"Version in en\""      
-#> [31] "  mainlanguage: 'en'"              "  languages: ['es', 'fr']"        
-#> [33] "title-es: title in es"             "title-fr: title in fr"            
-#> [35] "description-es: description in es" "description-fr: description in fr"
-#> [37] "author-es: author in es"           "author-fr: author in fr"          
-#> [39] "lang: en"
-fs::dir_tree(file.path(parent_dir, project_dir))
-#> /var/folders/gd/rrsyq__147x_8y_3zcpymj_40000gn/T//Rtmpr3nsvT/filed2be4d2b3e3/blop
-#> ├── _quarto.yml
-#> ├── about.es.qmd
-#> ├── about.fr.qmd
-#> ├── about.qmd
-#> ├── index.es.qmd
-#> ├── index.fr.qmd
-#> ├── index.qmd
-#> └── styles.css
-```
-
-``` r
-babelquarto::render_website(file.path(parent_dir, project_dir))
-```
-
-[Example](https://maelle.github.io/babelsite),
-[source](https://github.com/maelle/babelsite)
-
-## Configure the base URL
-
-Use the [usual Quarto
-field](https://quarto.org/docs/websites/website-tools.html), or use the
-`site_url` argument of `babelquarto::render_book()`.
-
-    book:
-      site-url: https://example.com
-
-## Configure the version text
-
-If you want the choice to be between, say “English” and “Español” rather
-than “Version in EN” and “Version in ES”, add these fields under the
-babelquarto YAML key, in `_quarto.yml`:
-
-``` yaml
-babelquarto:
-  languagecodes:
-  - name: es
-    text: "Español"
-  - name: en
-    text: "English"
-```
-
-Using `babelquarto::register_main_language()` and
-`babelquarto::register_further_languages()` will create the
-boilertemplate for these fields.
-
-## Configure the language link placement
-
-The language link menu can be placed in the *sidebar* or in the
-*navbar*. To change the placement, use the `languagelinks` field in the
-babelquarto YAML key, in `_quarto.yml`:
-
-``` yaml
-babelquarto:
-  languagelinks: sidebar
-```
-
-Beware: depending on the value of `languagelinks`, the corresponding
-menu needs to be configured. Books always have a *sidebar*, but websites
-can have either *navbar* or *sidebar*. See [Quarto
-documentation](https://quarto.org/docs/websites/website-navigation.html#side-navigation)
-for more information.
-
-## Distinct configuration per language
-
-There are two complementary/overlapping solutions for making the
-different language versions distinct (different banner, different navbar
-for instance):
-
-### Configuration file per language
+## Getting Started
 
-For a book with French (“fr”) and English (“en”) versions you could have
-three Quarto configuration files:
+The {babelquarto} package allows you to create a multilingual Quarto project (a book or a website).
```

Comment:
```suggestion
The {babelquarto} package allows you to create and render a multilingual Quarto project, book or website.
```

I try to avoid parentheses as I feel they break the flow somehow.

Reply:
Noted 😉